andela/wg-kalii
# #154953735 Exercise cache when deleting muscles

#### What does this PR do?
* Resets muscles cache after a muscle has been deleted

#### Description of Task to be completed?
* After deleting a muscle, it keeps appearing in descriptions which is probably due to the cache not being correctly reset; the cache should be reset after a muscle has been deleted.

#### How should this be manually tested?
After cloning the Repo, cd into it and activate the virtual environment. Install requirements into the virtual environment and run the following commands:

$./manage.py migrate
$./manage.py runserver

* Visit localhost:8000 from your browser and login into the application.
* Navigate to Excercises tab and click on the Muscles link and delete any Muscle
* Navigate to Exercises tab and click on the Muscles overview link (Any deleted muscle shouldn't appear here)

#### What are the relevant pivotal tracker stories?
#154953735
